src/components/Identity/createComponent.js

```diff
@@ -39,12 +38,6 @@ export default ({
       }
     },
     commands: {
-      syncIdentity: {
-        optionsValidator: validateSyncIdentityOptions,
-        run: options => {
-          return identityManager.sync(options.identity);
```

Comment:
So there is nothing in the identityManager that is specific of how the options were formatted in the user exposed command that we should remove as well?

Reply:
Great question. I think we should remove `identityManager.sync` as well since we won't be using it in the future.

Reply:
I mean we can even remove all the hashing and validation logic, and bring them back once we implement Phase 1 which adds a new config that specifies which Identity Namespaces the customer would like to mutate (Hash...)

@ninaceban What do you think?

Reply:
will remove them